scripts/linters/html_linter.py

```diff
@@ -117,6 +117,7 @@ def handle_starttag(self, tag, attrs):
             # Therefore the check should run only for those
             # attributes which have a value.
             if value:
+                value_in_quotes = True
```

Comment:
Do we need to define any default value for this variable before the if check?

Reply:
@DubeySandeep no, I haven't defined since the scope of value_in_quotes is inside the if statement. Should I do that?